src/Microsoft.Win32.Registry/tests/RegistryKey/RegistryKey_OpenSubKey_str_rkpc.cs

```diff
@@ -50,7 +50,6 @@ public void NegativeTests()
         }
 
         [Fact]
```

Comment:
I see, should we somehow mark the bug on GitHub as netfx-ported and mention which version it was ported to? cc @AlexGhiondea ... or is it overkill?